recipes/osgearth/all/conanfile.py

```diff
@@ -105,13 +105,13 @@ def requirements(self):
         self.requires("opengl/system")
         self.requires("gdal/3.4.3")
         self.requires("openscenegraph/3.6.5")
-        self.requires("libcurl/7.83.1")
+        self.requires("libcurl/8.2.0")
         self.requires("lerc/2.2")
         self.requires("rapidjson/1.1.0")
 
-        self.requires("zlib/1.2.12", override=True)
-        self.requires("libtiff/4.3.0", override=True)
-        self.requires("openssl/1.1.1l", override=True)
```

Comment:
Hi @mayeut, thanks for the contribution. One question,why is openssl removed? It was not needed?

Reply:
It was just an override.
With most recipes moving to `self.requires("openssl/>=1.1 <4")` (as is done in curl), the override is not required anymore.

Reply:
Oh, yes sorry, somehow I overlooked that 🤦